lib/src/main/java/growthbook/sdk/java/GBContext.java

```diff
@@ -46,6 +47,7 @@ public class GBContext {
     public GBContext(
             @Nullable String attributesJson,
             @Nullable String featuresJson,
+            @Nullable JsonObject features,
```

Comment:
Lines 49 and 50 look like the user of the library should pass `String` and `JsonObject`.
Seems like two constructors needed one with `String` and another one with `JsonObject`.
Do you allow us to try to add second constructor?

Reply:
yes, that makes sense.

Reply:
we might have to handle @Builder annotation given that works only on one constructor.

Reply:
It seems like tests were passed.

Reply:
If I remove `@Builder` annotation from the second constructor would it help?

Reply:
Yes, that would be better.
The primary interface should be the one with JsonObject.

Reply:
But without `featuresJson` typed with `String` we will need to correct all tests:
![image](https://github.com/growthbook/growthbook-sdk-java/assets/135710078/6f3e8998-8da1-46a9-9f2e-93646966bce9)

Reply:
```
JsonObject features = gson.fromJson(featuresJson, JsonObject.class);
GBContext context1 = GBContext.builder()
                            .features(features)
                            .attributesJson(getUserAttributesObj().toString()) // Optional
                            .trackingCallback(null)
                            .build();
GrowthBook growthBook = new GrowthBook(context1);
```
Right now with the code changes in the PR, the above way of initializing GB doesn't work, it leads to empty features jsonobject.

if we decide to not change the builder interface then we should remove `@Builder` from second constructor because it's allowing `features` to be set using the builder pattern but not using the value.
In case, anyone wants to pass features Json directly (instead of string) then either they'll have to use the constructor without builder or use the other builder and then pass feature Json separately.
Like this
```
 GBContext context1 = GBContext.builder()
                            .attributesJson(getUserAttributesObj().toString()) // Optional
                            .trackingCallback(null)
                            .build();
context1.setFeatures(features);
GrowthBook growthBook = new GrowthBook(context1);
```

Reply:
Seems like our intention to create the second constructor is wrong because it is not pure Java constructor but constructor annotated with Lombok Builder.

Reply:
The current changes looks good to me
let me know when can we merge this PR